Link to leaderboard and more on testnet points (#2981)
CodaProtocol/coda: frontend/website/docs/index.md

Trying out Coda is simple - head over to the [Getting Started page](/docs/getting-started/) to install the Coda client and join the network.

You can also sign up for the public testnet updates [here](https://bit.ly/TestnetForm).

<span id="testnet-points" />

> ## Testnet Points[\*](#disclaimer)

Reviewer: you don't need to do this, all headers automatically get tagged - this one will be `#testnet-points`

Author: I think there may be weirdness with the `\*` which is why I wanted to make it explicit just in case

> ## Testnet Points[\*](#disclaimer)

Reviewer 2: I don't think this parses correctly

Reviewer 3: lol it parses but looks a bit funky 
<img width="280" alt="Screen Shot 2019-07-24 at 6 22 03 PM" src="https://user-images.githubusercontent.com/1709621/61838846-0ebe4000-ae40-11e9-87ce-54ebe71eb1d3.png">

I don't think we need the link.
```suggestion
## Testnet Points*
```

> We want to recognize those who are actively involved in the network and our community. We’ll be issuing weekly challenges, tied to Testnet Points[\*](#disclaimer) to make it fun, interesting and a little competitive!

Reviewer: ```suggestion
The goal of Testnet Points is to recognize Coda community members who are actively involved in the network. There will be weekly challenges, tied to Testnet Points[\*](#disclaimer) to make it fun, interesting, and foster some friendly competition!
```

> You can track the community’s Testnet Points[\*](#disclaimer) in our public, [read-only leaderboard](http://bit.ly/TestnetBetaLeaderboard).

Reviewer: ```suggestion
You can track the Testnet Points[\*](#disclaimer) and current leaders in the public, [read-only leaderboard](http://bit.ly/TestnetBetaLeaderboard).
```

> Check out our [Discord](http://bit.ly/CodaDiscord) for specifics and updates on the challenges.

Reviewer: ```suggestion
Check out the [Discord server](http://bit.ly/CodaDiscord) and head to the #testnet-general channel for specifics and updates on challenges.
```
